config/application.properties.sample

```diff
@@ -64,6 +64,11 @@ server.context-path=/api/v1
 
 #Log settings for production. Change to log4j2.xml for dev
 logging.config=classpath:log4j2-prod.xml
+logging.file=logs/log.1
+
+#Spring Boot Actuator endpoint settings. Security through SecurityConfiguration.java
+management.security.enabled=false
```

Comment:
Actuator uses Spring Security and `management.security.role=ADMIN` configures it to use the `ADMIN` instead of `ACTUATOR` role. Maybe try pre-authenticating w/o basic auth? If that fails something might be broken in your security config.